src/react/components/common/condensedList/condensedList.tsx

```diff
@@ -15,8 +15,12 @@ interface ICondensedListProps {
     title: string;
     Component: any;
     items: any[];
+    displayEmptyMessage: boolean;
+    Toolbar?: any;
```

Comment:
Why split out the toolbar and the props?  If you want you can just accept a `JSX.element` here and would allow someone to specify their own rendering for this element and pass whatever props they want inline.